build_scripts/build_macos-2-installer.sh

```diff
@@ -16,7 +16,8 @@ echo "Chia Installer Version is: $CHIA_INSTALLER_VERSION"
 
 echo "Installing npm utilities"
 cd npm_macos || exit 1
-npm ci
+npm ci --foreground-scripts
```

Comment:
Did we want `--foreground-scripts` here?

```suggestion
npm ci
```

Reply:
This line was taken from Earl's PR: https://github.com/Chia-Network/chia-blockchain/pull/18171/files
so that this PR can replace it.
But you're right. The concept of 2 PRs are different. We shouldn't mix it up in one PR.